Refresh.GameServer/Endpoints/ApiV3/PhotoApiEndpoints.cs

```diff
@@ -16,6 +17,23 @@ namespace Refresh.GameServer.Endpoints.ApiV3;
 
 public class PhotoApiEndpoints : EndpointGroup
 {
+    [ApiV3Endpoint("photos/{id}", Method.Delete)]
+    [DocSummary("Deletes an uploaded photo")]
+    [DocError(typeof(ApiNotFoundError), "The photo cannot be found")]
+    [DocError(typeof(ApiValidationError), "The photo was not uploaded by you")]
+    // i dont know what type this response should be so im just gonna let you figure that out
+    public Response DeletePhoto(RequestContext context, GameDatabaseContext database, GameUser user, int id)
+    {
+        GamePhoto? photo = database.GetPhotoById(id);
+        if (photo == null) return NotFound;
+
+        if (photo.Publisher.UserId != user.UserId)
+            return Forbidden;
+        
+        database.RemovePhoto(photo);
+        return OK;
```

Comment:
You shouldn't return status codes directly with APIv3, the `ApiError` system will manage that.